makefile

```diff
@@ -208,13 +208,17 @@ test-names = uexpr-tests adt-tests type-tests eval-tests show-tests \
 
 lib-names = diagram plot png
 
-all-names = $(test-names:%=tests/%) $(example-names:%=examples/%)
+text-names = functions
```

Comment:
Because "doc" is already being used in the makefile for generated targets -- doc-lib-names, doc-example-names, etc.  doc-text-names seemed better than doc-doc-names.

Reply:
Sure, but we can change the names of `doc` targets! And `doc` is just a _significantly_ more conventional directory name for docs than "texts". Discoverability matters!

Reply:
As discussed off-thread, will make this change in a separate PR.